src/graphqelm.ts

```diff
@@ -35,14 +36,40 @@ if (!graphqlUrl) {
 }
 const tsDeclarationPath = args.output
 
+const writeWithElmFormat = (path: string, value: string): void => {
+  const elmFormat = spawn('./node_modules/.bin/elm-format', [
+    '--stdin',
+    '--output',
+    path
+  ])
+
+  elmFormat.stdin.write(value)
+  elmFormat.stdin.end()
+
+  elmFormat.stdout.on('data', data => {
+    console.log(data.toString())
+  })
+
+  elmFormat.stderr.on('data', data => {
+    console.log(data.toString())
```

Comment:
If elm-format cannot auto format the file - the error message will be output to the terminal.